scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -23,41 +27,90 @@
+    for sensor_chart_version in sensor_chart_versions
+]
 # Latest sensor vs central versions in central_chart_versions
-test_tuples.extend([Chart_versions(central_version=central_chart_version, sensor_version=latest_tag) for central_chart_version in central_chart_versions])
+test_tuples.extend(
+    [
+        ChartVersions(central_version=central_chart_version,
+                      sensor_version=latest_tag)
+        for central_chart_version in central_chart_versions
+    ]
+)
 
-# Support exception for latest central and sensor 3.74 as per https://issues.redhat.com/browse/ROX-18223
-support_exceptions = [Chart_versions(central_version=latest_tag, sensor_version=get_latest_helm_chart_version_for_specific_release("stackrox-secured-cluster-services", Release(major=3, minor=74)))]
+# Support exception for latest central and sensor 3.74 as per
+# https://issues.redhat.com/browse/ROX-18223
+support_exceptions = [
+    ChartVersions(
+        central_version=latest_tag,
+        sensor_version=get_latest_helm_chart_version_for_specific_release(
+            "stackrox-secured-cluster-services", Release(major=3, minor=74)
+        ),
+    )
+]
 
-test_tuples.extend(support_exception for support_exception in support_exceptions if support_exception not in test_tuples)
+test_tuples.extend(
+    support_exception
+    for support_exception in support_exceptions
+    if support_exception not in test_tuples
+)
 
-gkecluster = GKECluster("compat-test", machine_type="e2-standard-8", num_nodes=2)
+gkecluster = GKECluster(
+    "compat-test", machine_type="e2-standard-8", num_nodes=2)
 
 failing_tuples = []
-for tuple in test_tuples:
-    os.environ["CENTRAL_CHART_VERSION_OVERRIDE"] = tuple.central_version
-    os.environ["SENSOR_CHART_VERSION_OVERRIDE"] = tuple.sensor_version
+for test_tuple in test_tuples:
+    os.environ["CENTRAL_CHART_VERSION_OVERRIDE"] = test_tuple.central_version
+    os.environ["SENSOR_CHART_VERSION_OVERRIDE"] = test_tuple.sensor_version
     try:
         make_compatibility_test_runner(cluster=gkecluster).run()
     except Exception as e:
-        print(f"Exception \"{str(e)}\" raised in compatibility test for central version {tuple.central_version} and sensor version {tuple.sensor_version}",
-            file=sys.stderr)
-        failing_tuples.append(tuple)
+        print(
+            f'Exception "{str(e)}" raised in compatibility test for '
+            f'central version {test_tuple.central_version} and '
+            f'sensor version {test_tuple.sensor_version}',
+            file=sys.stderr,
+        )
+        failing_tuples.append(test_tuple)
 
 if len(failing_tuples) > 0:
-    failing_string = ', '.join([("(Central v" + str(failing_tuple.central_version) + ", Sensor v" + str(failing_tuple.sensor_version) + ")") for failing_tuple in failing_tuples])
-    raise RuntimeError("Compatibility tests failed for versions " + failing_string + ".")
+    # pylint: disable=invalid-name
```

Comment:
What's the invalid name here?
BTW the fact that `ChartVersions` is a (named) tuple is an implementation detail, so personally I'd avoid the `_tuple` in variable names. It's also a bit confusing, since there's also the unnamed tuple type, which springs to mind first when one sees "tuple". And seeing a tuple with a field selector rather than index subscript is odd.

Reply:
> What's the invalid name here? 
`failing_string` - pylint wanted me to UPPER_CASE it, which just seemed wrong.

> BTW the fact that `ChartVersions` is a (named) tuple is an implementation detail, so personally I'd avoid the `_tuple` in variable names. It's also a bit confusing, since there's also the unnamed tuple type, which springs to mind first when one sees "tuple". And seeing a tuple with a field selector rather than index subscript is odd.
Good point. The only real change I made here was tuple -> test_tuple because the use of the reserved word (tuple) caused pylint to crash.

Reply:
> `failing_string` - pylint wanted me to UPPER_CASE it, which just seemed wrong.

I think I know why - because all this code is on the top level, rather than in a function, lint thinks we're defining a constant...